installer/sharch_body.sh

```diff
@@ -27,6 +27,7 @@ echo " OK."
 
 # Untar and launch install script in a tmpfs
 cur_wd=$(pwd)
+export cur_wd
```

Comment:
cur_wd is the absolute path prefix prior to the user's target. say, /home/padman/test/sonic-buildimage.  Since the installer (in a "build" mode) does not have a real device to mount, it will fallback to /tmp/tmp... (based on mktmp -d). When the final sonic-broadcom.bin is created under /tmp... build_image.sh cannot correctly extract the sonic-broadcom.bin if there are parallel builds on the same host...

One other way would be to pass the cur_wd equivalent to the installer - but that did not appear to be clean. 

Exporting cur_wd will ensure that the image gets built only in the specific target dir where the build was invoked.